package.json

```diff
@@ -80,32 +80,31 @@
     "@babel/plugin-transform-object-assign": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "convert-source-map": "^2.0.0",
-    "invariant": "^2.2.4",
-    "string-hash-64": "^1.0.3"
+    "invariant": "^2.2.4"
```

Comment:
Fun fact: we use `invariant` only in one place [here](https://github.com/software-mansion/react-native-reanimated/blob/62fef6f657d638de5dcc80e13f420593eee9aea7/src/createAnimatedComponent.tsx#L246-L251) and technically we could just embed it's code as you did with `string-hash-64` but it doesn't make sense to do so because `invariant` is already a dependency of `react-native` so it will be installed anyway.

Reply:
The real reason for me to embed that code is that it didn't have `@types` package on npm 😆, but that is indeed a fun fact.